DriverLoader.h

```diff
@@ -40,14 +40,28 @@
 namespace osvr {
 namespace vive {
     struct CouldNotLoadDriverModule : std::runtime_error {
+#if defined(OSVR_LINUX) || defined(OSVR_MACOSX)
```

Comment:
I don't know, is Windows can give a error string. 
But yes, there is can be a checking for NULL pointer. :)